backend/workflow_manager/endpoint/static/src/file.json

```diff
@@ -4,17 +4,21 @@
     "type": "object",
     "required": [],
     "properties": {
-        "rootFolder": {
-            "type": "string",
-            "title": "Folder to process",
-            "default": "",
-            "description": "The root folder to start processing files from. Leave it empty to use the root folder"
+        "folders": {
+            "type": "array",
+            "title": "Folders to process",
+            "description": "Folders in the connected filesystem to start processing files from. Leave it empty or pass '/' to specify the root folder.",
```

Comment:
I think , it would be better if we describe how to list the folders, By comma or space etc..

Reply:
@muhammad-ali-e this is similar to the array input of classification bins - will anyway update the description